config.lua.dist

```diff
@@ -98,6 +99,7 @@ forceMonsterTypesOnLoad = true
 cleanProtectionZones = false
 showPlayerLogInConsole = true
 checkDuplicateStorageKeys = false
+expDisplayRates = true
```

Comment:
Is a configuration value really required for this feature?

Reply:
I don't think so, should always be enabled